lib/rentlinx/validators/attribute_processor_service.rb

```diff
@@ -0,0 +1,31 @@
+module Rentlinx
+  class AttributeProcessor
```

Comment:
I just tried it and it seems as though it doesn't play well with the test suite or something. If you change `Rentlinx::AttributeProcessor` to `Rentlinx::Validator::AttributeProcessor`, it fails as a single test, but not when you run the whole file.